mod/dfrn_notify.php

```diff
@@ -133,7 +136,7 @@ function dfrn_notify_post(App $a) {
 
 	}
 
-
+	/// @TODO remove this old-lost code then?
```

Comment:
done! :-) I better let you [with Friendica more experienced developers] a chance to review it rather than instant-removing it.